util/tlgen/item.py

```diff
@@ -36,7 +36,9 @@ class Node:
     # Other nodes have this fields set to -1.
     address_from: int = -1
     address_to: int = -1
+    addr_ranges = Dict[int, Tuple[int, int]]
```

Comment:
Should this be
`addr_ranges: Dict[int, Tuple[int, int]] = {}`

But more curious is why these classes declare both instance and class variables. It would seem the class variables are unused. Was this code just created by @a-will or are you familiar enough with it?

Reply:
Oh yes, that's correct! 

This behavior is due to Python's typing system. The variables shown above are instance variables by default, not class variables. To explicitly declare a class variable with type hints, you need to use the `ClassVar` annotation.

I think the following documentation covers this:
https://typing.readthedocs.io/en/latest/spec/class-compat.html